gcs-action/.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: 'build-test'
```

Comment:
Please rename this file to match the convention of the existing actions.

Reply:
Done